starknet_py/net/account/account_client.py

```diff
@@ -493,9 +493,6 @@ async def estimate_fee(
             block_number=block_number,
         )
 
-    async def get_code(self, *args, **kwargs):
```

Comment:
This is a breaking change. I suggest moving the error here if wrong client is used.